spec/services/flow/steps/addresses/correspondence_address_lookups_step_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.describe Flow::Steps::Addresses::CorrespondenceAddressLookupsStep, type: :request do
```

Comment:
Do these new step tests need `, type: :request`?

Reply:
I added this in because without it, the '_path' helpers didn't work